orbit/cmd/orbit/orbit.go

```diff
@@ -870,11 +870,6 @@ func main() {
 			orbitClient.RegisterConfigReceiver(update.ApplyNudgeConfigReceiverMiddleware(update.NudgeConfigFetcherOptions{
 				UpdateRunner: updateRunner, RootDir: c.String("root-dir"), Interval: nudgeLaunchInterval,
 			}))
-			if orbitClient.GetServerCapabilities().Has(fleet.CapabilityEscrowBuddy) {
```

Comment:
Is this the fix for the issue Gabe found? Is it that we haven't initiated the capabilities checker yet?

Reply:
that's right, sorry for not being clear about it.